tests/test_albertheijn.py

```diff
@@ -1,3 +1,4 @@
+# canonical_url : https://www.ah.nl/allerhande/recept/R-R1198767/rijkgevulde-vegan-pastasalade
```

Comment:
Ok, I understand what you're doing with this now that I see it in action!

I have a suggestion to encode it in a way that might be easier to re-activate as test coverage in future: could you write it as a `test_canonical_url` method but add the [`unittest.skip` decorator](https://docs.python.org/3/library/unittest.html#unittest.skip) to it?